File: DataGenerator/domain/group.py


class Group:

    def __init__(self, id, facultyId, year):
        self.__id = id
        self.__facultyId = facultyId
        self.__year = year

    @property
    def id(self):
        return self.__id

    @property
    def facultyId(self):
        return self.__facultyId

    @property
    def year(self):
        return self.__year

    def __str__(self):
        return str(self.__id) + ", " + str(self.__facultyId) + ", " + str(self.__year)

File: AcademicWebApp/src/main/java/com/example/AcademicWebApp/Models/Group.java
package com.example.AcademicWebApp.Models;

import lombok.*;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Data
@Entity(name = "group")

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Group {

    @Id
    @Column(name = "gid")
    private Integer gid;
    @Column(name = "faculty")
    private Integer faculty;
    @Column(name = "year")
    private Integer year;



}